kdeinit5: preparing to launch '/usr/lib/kf5/klauncher'
kdeinit5: Launched KLauncher, pid = 612, result = 0
kdeinit5: opened connection to :0
kdeinit5: Got SETENV 'GTK_RC_FILES=/etc/gtk/gtkrc:/home/govi/.gtkrc:/home/govi/.config/gtkrc' from launcher.
kdeinit5: Got SETENV 'GTK2_RC_FILES=/etc/gtk-2.0/gtkrc:/home/govi/.gtkrc-2.0:/home/govi/.config/gtkrc-2.0' from launcher.
kdeinit5: Got SETENV 'XCURSOR_THEME=GoVi-Cu' from launcher.
kdeinit5: Got SETENV 'XCURSOR_SIZE=24' from launcher.
kdeinit5: Got SETENV 'SESSION_MANAGER=local/GoVi-PC:@/tmp/.ICE-unix/658,unix/GoVi-PC:/tmp/.ICE-unix/658' from launcher.
kdeinit5: Got SETENV 'SESSION_MANAGER=local/GoVi-PC:@/tmp/.ICE-unix/658,unix/GoVi-PC:/tmp/.ICE-unix/658' from launcher.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/desktop.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/desktop.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
imwheel: no process found
INFO: imwheel started (pid=907)
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/tags.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/tags.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/trash.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/trash.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: PID 1992 terminated.
kdeinit5: PID 2001 terminated.
kdeinit5: PID 2002 terminated.
kdeinit5: PID 1998 terminated.
kdeinit5: PID 1993 terminated.
kdeinit5: PID 2004 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/tags.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/tags.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/trash.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/trash.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 2117 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 2118 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: PID 2125 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 2581 terminated.
kdeinit5: PID 2693 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 2788 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 2891 terminated.
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
2021-01-15 23:38:51,774 [    313]   WARN - llij.ide.plugins.PluginManager - Resource bundle redefinition for plugin 'com.jetbrains.pycharm.community.customization'. Old value: messages.ActionsBundle, new value: messages.PyBundle 
2021-01-15 23:38:53,486 [   2025]   WARN - nSystem.impl.ActionManagerImpl - keymap "Visual Studio" not found [Plugin: com.intellij] 
2021-01-15 23:38:53,489 [   2028]   WARN - nSystem.impl.ActionManagerImpl - keymap "Eclipse" not found [Plugin: com.intellij] 
2021-01-15 23:38:53,489 [   2028]   WARN - nSystem.impl.ActionManagerImpl - keymap "NetBeans 6.5" not found [Plugin: com.intellij] 
2021-01-15 23:38:53,489 [   2028]   WARN - nSystem.impl.ActionManagerImpl - keymap "Visual Studio" not found [Plugin: com.intellij] 
2021-01-15 23:38:54,105 [   2644]   WARN - nSystem.impl.ActionManagerImpl - keymap "Eclipse" not found [Plugin: PythonCore] 
2021-01-15 23:38:54,105 [   2644]   WARN - nSystem.impl.ActionManagerImpl - keymap "NetBeans 6.5" not found [Plugin: PythonCore] 
2021-01-15 23:38:58,150 [   6689]   WARN - openapi.wm.impl.ToolWindowImpl - ToolWindow icons should be 13x13. Please fix ToolWindow (ID:  Problems View) or icon jar:file:/home/govi/Programas/Pycharm/lib/icons.jar!/general/warning.svg 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - Some skeletons failed to generate 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtBluetooth 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtNetworkAuth 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtNfc 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtQuick3D 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtRemoteObjects 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtSerialPort 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtWebSockets 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - PyQt5.QtXmlPatterns 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - _decimal 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - boost.mpi 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - cephfs 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - imathnumpy 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - imobiledevice 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - lirc._client 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - rados 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - rbd 
2021-01-15 23:39:03,135 [  11674]   WARN - .skeletons.PySkeletonRefresher - rgw 
kdeinit5: PID 2113 terminated.
kdeinit5: PID 882 terminated.
kdeinit5: PID 874 terminated.
kdeinit5: PID 1227 terminated.
kdeinit5: PID 1221 terminated.
kdeinit5: PID 1226 terminated.
kdeinit5: PID 2112 terminated.
kdeinit5: Got EXEC_NEW 'onboard' from launcher.
kdeinit5: preparing to launch 'libkdeinit5_onboard'
Could not open onboard using a library: Cannot load library libkdeinit5_onboard: (libkdeinit5_onboard: no se puede abrir el fichero del objeto compartido: No existe el fichero o el directorio)
23:41:10.359 WARNING Config: mousetweaks GSettings schema not found, mousetweaks integration disabled.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 2743 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: PID 8001 terminated.
kdeinit5: PID 2114 terminated.
kdeinit5: PID 6793 terminated.

(liferea:8299): libsecret-CRITICAL **: 23:42:39.223: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.224: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.226: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.228: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.229: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.231: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.232: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.234: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.235: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.237: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.239: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.240: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.242: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.244: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.247: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.249: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.252: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.254: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.264: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.270: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.271: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.273: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.274: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.276: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.277: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.279: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.280: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.282: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.283: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.285: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.286: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.288: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.290: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.291: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.292: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.294: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.296: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.297: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.298: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.300: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.302: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.303: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.305: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.306: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.308: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.309: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.311: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.312: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.314: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.315: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.317: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.319: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.320: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.322: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.323: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.325: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.327: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.328: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.330: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.331: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.333: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.334: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.335: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.337: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.339: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.340: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.342: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.343: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.345: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.346: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.347: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.349: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.351: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.352: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.354: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.355: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.357: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.358: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.360: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.361: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.363: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.365: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.366: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.368: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.369: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.371: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.372: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.374: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.375: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.377: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.378: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.379: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.381: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.382: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.384: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.385: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.386: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.387: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.388: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.389: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.391: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.393: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.394: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.395: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.396: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.398: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.398: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.400: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.401: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.402: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.403: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.404: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.405: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.407: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'

(liferea:8299): libsecret-CRITICAL **: 23:42:39.408: secret_service_load_collections_sync: assertion 'paths != NULL' failed
Traceback (most recent call last):
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 54, in do_query
    coll = self.get_collection()
  File "/usr/lib/liferea/plugins/gnome-keyring.py", line 44, in get_collection
    colls = service.get_collections()
AttributeError: 'NoneType' object has no attribute 'get_collections'
kdeinit5: PID 8132 terminated.
kdeinit5: Got EXEC_NEW 'onboard' from launcher.
kdeinit5: preparing to launch 'libkdeinit5_onboard'
Could not open onboard using a library: Cannot load library libkdeinit5_onboard: (libkdeinit5_onboard: no se puede abrir el fichero del objeto compartido: No existe el fichero o el directorio)
23:43:19.942 WARNING Config: mousetweaks GSettings schema not found, mousetweaks integration disabled.
